src/components/Modal.js

```diff
@@ -53,7 +53,7 @@ const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCance
       nodeToFocus.focus()
     },
     style: { overlay: styles.overlay, content: { ...styles.modal, width } },
-    aria: { labelledby: titleId },
+    aria: { labelledby: titleId, modal: true },
```

Comment:
Adds the `aria-modal` attribute, indicating to screen readers that they should stay within the dialog and not read the rest of the page while the modal is active. This is still emergent, but has reasonable screen reader support at this point. https://a11ysupport.io/tech/aria/aria-modal_attribute